zabbix/map.jinja

```diff
@@ -149,6 +149,20 @@
     },
   },
 
+  'Windows':{
+    'user': 'Administrator',
+    'group': 'Administrators',
+    'agent': {
+      'version': '3.0.28.2400',
```

Comment:
Does this particular version is required? Can it be specified like `3.0` or `3.0.*` to leave more room for updates? But overall it's not very important.